icons/trending-flat.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path
+    d="m2 12 1.651-.99A4.56 4.56 0 0 1 9.5 12v0a4.56 4.56 0 0 0 5.849.99l.43-.257A5.136 5.136 0 0 1 18.425 12H22" />
```

Comment:
The tail feels top-heavy.

Based on wave ![icon - 2023-11-05T190744 857](https://github.com/lucide-icons/lucide/assets/25524993/8d7cd789-699e-44ff-8f8d-9ea1886a28ae)

```suggestion
  <path d="M2 12c.6.5 1.2 1 2.5 1 2.5 0 2.5-2 5-2 2.6 0 2.4 2 5 2 2.5 0 2.5-2 7.5-1" />
```